docs/topics/multiplatform/multiplatform-native-artifacts.md

```diff
@@ -0,0 +1,370 @@
+> is a debug test executable that lets you run unit tests from the `test` compilation.
+>
+{type="note"}
+
+## Declare binaries
+
+Use the following kinds of binaries to declare elements of the `kotlinArtifacts` DSL: 
+
+| Factory method | Binary kind             | Available for                                |
+|----------------|-------------------------|----------------------------------------------|
+| `sharedLib`    | Shared native library   | All native targets, except for `WebAssembly` |
+| `staticLib`    | Static native library   | All native targets, except for `WebAssembly` |
+| `framework`    | Objective-C framework   | macOS, iOS, watchOS, and tvOS targets only   |
+| `fatFramework` | Universal fat framework | macOS, iOS, watchOS, and tvOS targets only   |
+| `XCFramework`  | XCFramework framework   | macOS, iOS, watchOS, and tvOS targets only   |
+
+The simplest version requires the `target` (or `targets`) and `modes` parameters for the selected build type. Currently, 
+two build types are available: 
+
+* `DEBUG` – produces a non-optimized binary with debug information 
+* `RELEASE` – produces an optimized binary without debug information
+
+You can specify for which build types to create binaries. In the following example, both `debug` and `release`
+executable binaries are created:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+kotlin {
+    kotlinArtifacts {
+        Native.Library {
+            target = iosX64 // Define your target instead
+            modes(DEBUG, RELEASE)
+            // Binary configuration
+        }
+    }
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlin {
+    kotlinArtifacts {
+        it.native.Library {
+            target = iosX64 // Define your target instead
+            modes(DEBUG, RELEASE)
+            // Binary configuration
+        }
+    }
+}
+```
+
+</tab>
+</tabs>
+
+You can also declare binaries with custom names:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+kotlin {
+    kotlinArtifacts {
+        Native.Library("mylib") {
+            // Binary configuration
+        }
+    }
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlin {
+    kotlinArtifacts {
+        it.native.Library("mylib") {
+            // Binary configuration
+        }
+    }
+}
+```
+
+</tab>
+</tabs>
+
+The argument sets a name prefix, which is the default name for the binary file. For example, for Windows the code 
+produces the files `mylib.dll`.
+
+## Build binaries for several modules
+
+If your project has multiple Kotlin modules, and you need to access them from your iOS app, you'll encounter an issue
+− the usage of several Kotlin/Native modules in Swift is limited. With Kotlin artifact DSL, you can export multiple
+Kotlin modules into a single artifact.
```

Comment:
If your project has multiple Kotlin modules, and you need to access them from your iOS app, you'll encounter an issue − the usage of several Kotlin/Native **artifacts** in Swift is limited.